### PR TITLE
added icons to tabs

### PR DESCRIPTION
**Pull request by scottaudet**

### CODE REVIEW

File: ApolloFitness/navigation/MainTabNavigator.js [hunk 1]
```diff
@@ -23,14 +24,7 @@ const HomeStack = createStackNavigator(
...
   tabBarIcon: ({ focused }) => (
-    <TabBarIcon
-      focused={focused}
-      name={
-        Platform.OS === 'ios'
-          ? `ios-information-circle${focused ? '' : '-outline'}`
-          : 'md-information-circle'
-      }
-    />
+    <TabBarIcon focused={focused} name={Platform.OS === 'ios' ? 'ios-home': 'md-home'}/>
```
Comment (isaacsparkswilley) [inline ApolloFitness/navigation/MainTabNavigator.js]: Do we need to check for iOS here since we're just planning on building to android?

Comment (jacob-morin-dev) [reply ApolloFitness/navigation/MainTabNavigator.js]: we don't need to, technically we're wasting a few bytes to check, however is it good form and makes the program a little more future proof if we ever decide to build iOS later

**Comment by scottaudet** [comment]
Removed the labels. Should be good to go.

**Comment by scottaudet** [comment]
Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-18 at 20 30 25

**Pull request merged**